library/http_client_extension/src/request/request.e

```diff
@@ -0,0 +1,199 @@
+note
```

Comment:
I guess I'd like to see the benefit here

code with http_client  vs code with http_client_extension